admin/src/Accounts/AccountList.tsx

```diff
@@ -1,11 +1,23 @@
-import { Datagrid, List, TextField } from "react-admin";
+import {
+	Datagrid,
+	List,
+	NumberField,
+	ReferenceManyCount,
+	TextField,
+} from "react-admin";
 
 export function AccountList() {
 	return (
 		<List resource="accounts">
 			<Datagrid>
```

Comment:
I know the current guessers use `Datagrid` but we now promote `DataTable` (https://marmelab.com/react-admin/DataTable.html) instead and I missed this in the previous PR.